get-started/quickstart.rst

```diff
@@ -124,6 +124,9 @@ want to monitor traffic on):
 
      $ zeek -i en0 -C
 
+Commands which use a network device require root access in order to monitor
+traffic.
```

Comment:
This wording confused me a little at first. I switched around the ordering a little, plus root access *may* be required but not always.

```suggestion
Root access is typically required to run commands which monitor a network device.
```